supported-standards/lens-metadata-standards/publication/event/1.0.0/README.md

```diff
@@ -0,0 +1,128 @@
+# Event 1.0.0
+
+## JSON schema
+
+The JSON schema can be found [here](./schema.json).
+
+## TypeScript
+
+References:
+
+- [MarketplaceMetadata](../../shared-ts-interfaces/marketplace-metadata.ts)
+- [MetadataAttribute](../../../shared-ts-interfaces/metadata-attribute.ts)
+- [MetadataCommon](../../shared-ts-interfaces/metadata-common.ts)
+- [PublicationMainFocus](../../shared-ts-interfaces/publication-main-focus.ts)
+- [MediaVideo](../../shared-ts-interfaces/media/media-video.ts)
+- [MediaImage](../../shared-ts-interfaces/media/media-image.ts)
+- [MediaAudio](../../shared-ts-interfaces/media/media-audio.ts)
+
+```ts
+interface EventMetadata extends MarketplaceMetadata {
+  /**
+   * The JSON schema for the event
+   */
+  $schema: "https://raw.githubusercontent.com/lens-protocol/LIPs/feat/metadata-standards/lens-metadata-standards/publication/event/1.0.0/schema.json";
+
+  /**
+   * The metadata details for the event
+   */
+  lens: EventMetadataDetails;
+
+  /**
+   * If you use an appId and you only want your app to be able to publish under it,
+   * you will need to sign the `lens` object with your server private key
+   * and populate the signature here. We will need your public key to verify this.
+   * If the signature does not match what we have in our mappings it will not be surfaced
+   * in the API.
+   */
+  signature?: Signature;
+}
+
+interface EventMetadataDetails extends MetadataCommon {
```

Comment:
I'd add an organizer fields, so you can create/list multiple events.
I guess by default that would be the publication owner / lens profile Id.
But you might want to add or list existing events for others.

Reply:
> you might want to add or list existing events for others

How do you see this working? Like someone else delegated to create events in behalf of an group/organization?

In these terms it seems more of an operational thing that would fall under the Lens v2 Profile Manager feature.
In Lens v2 a Lens Profile owner can delegate to other addresses ability to do social actions (post/comments/collect/follow) in behalf of their profile.

In this example the even owner would be the Profile ID of the delegated profile and not related to the address used to publish the event.

Does this fit into the picture you had in mind for this field?

Reply:
I was thinking about it from a curator perspective. E.g. larger blockchain event weeks (e.g. Devconnect, EthCC week, etc.) might want to add and list events which aren't organized 'owned' by them. 
Maybe a checkbox could help to indicate if the creator of an event is also the organizer or just listing it.

Altho that might open some complexities if the actual owner wants to claim it later?

Maybe just Lens profiles makes most sense for now.

Reply:
Another suggestion here is to add an optional `speakers` field, which could reference Lens profiles.